packages/native-stack/src/types.tsx

```diff
@@ -31,6 +31,10 @@ export type NativeStackNavigationEventMap = {
    * Event which fires when a transition animation ends.
    */
   transitionEnd: { data: { closing: boolean } };
+  /**
+   * Event which fires when a header height gets changed.
+   */
+  headerHeightChange: { data: { headerHeight: number } };
```

Comment:
So we're not using that event anymore?

Reply:
Yep, I've exposed this event type if anyone would like to use it, but tbh I don't think if anyone would like to emit it 😅

Reply:
Removed in https://github.com/react-navigation/react-navigation/pull/11663/commits/d075678ee60d13ec2d4f2615dd665566c7d37a03 👍